Sources/SmileID/Classes/SelfieCapture/ViewModel/SelfieCaptureViewModel.swift

```diff
@@ -404,6 +407,11 @@ final class SelfieCaptureViewModel: ObservableObject {
     }
 
     func submit() {
+        if !shouldSubmitJob {
+            try? LocalStorage.delete(at: files)
```

Comment:
Is this attempting to delete the files if we shouldn't submit the job? How would that work with document verification, since we'll still need the selfie file for submission?

Reply:
Yes exactly, this is deleting the files if we aren't submitting a Smart Selfie Auth or Enrolment. The view model saves the files to disk as we capture them and also in memory. We delete the files and passed to the images in memory to the document capture flow. This will be reworked when the result methods return paths to the files

Reply:
> This will be reworked when the result methods return paths to the files

Ack